spacelift/resource_stack_dependency_reference_test.go

```diff
@@ -0,0 +1,81 @@
+package spacelift
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	. "github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/testhelpers"
+)
+
+func TestStackDependencyReferenceResource(t *testing.T) {
+	const resourceName = "spacelift_stack_dependency_reference.test"
+
+	t.Run("creates and updates stack dependency reference", func(t *testing.T) {
```

Comment:
Could you add a test for deletion as well?

Reply:
added